README.md

```diff
@@ -1,3 +1,34 @@
-Icon from:
+# sIndicator
+## What is sIndicator?
+sIndicator is a web extension distributed individually to syndication partners (i.e. not available through the store) to allow them to determine whether a given piece of content that they are viewing is syndicatable or not.
+In this context, we define syndicatable as
+`rights.syndicable=="true"`
+in CAPI, for a given CAPI id
+The tool does not provide client-specific, contract-specific knowledge of what content is available: it only provides a generic indication of whether a piece of content is syndicatable.
```

Comment:
Markdown only adds a break between paragraphs when there’s a blank line, and I think this should probably be spaced like this:

```suggestion
sIndicator is a web extension distributed individually to syndication partners (i.e. not available through the store) to allow them to determine whether a given piece of content that they are viewing is syndicatable or not.

In this context, we define syndicatable as `rights.syndicable=="true"` in CAPI, for a given CAPI id.

The tool does not provide client-specific, contract-specific knowledge of what content is available: it only provides a generic indication of whether a piece of content is syndicatable.
```